src/bitcoind/interface.rs

```diff
@@ -89,64 +86,46 @@ impl BitcoinD {
             node_client,
             watchonly_client,
             cpfp_client,
+            try_limit: 1,
```

Comment:
This should be documented above that it will set the number of retries to 0 and the caller should use the introduced API to set it.